### DISCUSSION TITLE
Bedava 3000 puan IBAX nasıl kazanılır?

### DISCUSSION THREAD

**Discussion by arfdgn** [IBAX-io/go-ibax#822] [Soru Cevap - Türkçe]
Github uygulamasını Play Storeden indirdikten sonra kayıt olun. Daha sonra İbax girip gösterge panelinden Github linkine tıklayın. Star ve Fork yazan yere tıklayıp ibax geri dönün. Finishede tıklayın. 3000 puan alacaksınız.

**Comment by Yusufknk** [comment]
Aynı zamanda web sitesi üzerinden de yapabilirsiniz.

**Reply by DiamondG3m** [reply]
Hi
